configs/benchmark-ps.json

```diff
@@ -53,13 +53,13 @@
     "dashboard": false,
     "crosstx": 30,
     "attacked_mode": 0,
-    "peer_per_shard": 10,
-    "minpeer": 6,
+    "peer_per_shard": 15,
+    "minpeer": 8,
```

Comment:
use 3 for minpeer, we only have 5 nodes, minpeer is at node level.

Reply:
Updated to 3